### PR TITLE
add check to ensure versions match

### PR DESCRIPTION
**Pull request by robrap**
Comments state that the python and npm version should match. However,
comments are easy to miss. Adds a script to check.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "@edx/edx-proctoring",
   "//": "Be sure to update the version number in edx_proctoring/__init__.py",
   "//": "Note that the version format is slightly different than that of the Python version when using prereleases.",
-  "version": "2.1.9",
+  "version": "2.1.10",
```
Comment (robrap) [inline package.json]: **Reviewer Note:** Temporary mismatch to manually test before reverting this line.

File: tox.ini [hunk 2]
```diff
@@ -16,7 +17,7 @@ deps =
     drflatest: djangorestframework
     -rrequirements/test.txt
 commands =
-    py.test -rfe --cov=edx_proctoring --cov-report=html --ds=test_settings -n 3 {posargs}
+    py.test -rfe --cov=edx_proctoring --cov-report=html --ds=test_settings {posargs:-n 3}
```
Comment (robrap) [inline tox.ini]: **Reviewer Note:** Unrelated fix to allow `--pdb` to work.

File: edx_proctoring/scripts/version_check.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+import inspect
+import os
+import sys
+from edx_proctoring import __version__ as python_version
+
+script_dir = os.path.dirname(os.path.abspath(inspect.getfile(inspect.currentframe()))) # script directory
+package_json_filename = os.path.join(script_dir, '..', '..', 'package.json')
+
+with open(package_json_filename) as f:
+    package_json_expected_version = '"version": "{}"'.format(python_version)
```
Comment (feanil) [inline edx_proctoring/scripts/version_check.py]: I think you should load this as JSON instead of reading the stream.
